app/src/main/java/com/example/javacodersnairobi/model/GithubUsers.java

```diff
@@ -0,0 +1,47 @@
+package com.example.javacodersnairobi.model;
+
+import com.google.gson.annotations.SerializedName;
+
+public class GithubUsers {
+    @SerializedName("avatar_url")
+    String avatar;
+
+    @SerializedName("name")
+    String username;
+
+    @SerializedName("bio")
```

Comment:
I believe you can omit using the `@SerializedName` annotation in cases where the returned field is the same as the model property name. You can the model as 
```
public class GithubUsers {
    @SerializedName("avatar_url")
    String avatar;

    @SerializedName("name")
    String username;

    String bio;

    @SerializedName("public_repos")
    String repos;

    String followers;

    String following;

  ............```